docs/learn/learn-treasury.md

```diff
@@ -188,32 +194,22 @@ why Bob deserves the tip.
 As mentioned above, Alice must also lock up a deposit for making this report. The deposit is the
 base deposit as set in the chain's parameter list, plus the additional deposit per byte contained in
 the reason. This is why Alice submitted a URL as the reason instead of the explanation directly: it
-was cheaper for her to do so.
-
-For her trouble, Alice is able to claim the eventual finder's fee if the tip is approved by the
+was cheaper for her to do so. For her trouble, Alice is able to claim the eventual finder's fee if the tip is approved by the
 tippers.
 
 Since the tipper group is the same as the Council, the Council must now collectively (but also
-independently) decide on the value of the tip that Bob deserves.
-
-Charlie, Dave, and Eve all review the report and make tips according to their personal valuation of
-the benefit Bob has provided to Kusama.
-
-For example:
-
-Charlie tips {{ polkadot: 10 DOT :polkadot }}{{ kusama: 1 KSM :kusama }}. Dave tips
-{{ polkadot: 30 DOT :polkadot }}{{ kusama: 3 KSM :kusama }}. Eve tips
+independently) decide on the value of the tip that Bob deserves. Charlie, Dave, and Eve all review the report and make tips according to their personal valuation of
+the benefit Bob has provided to {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}. Charlie tips {{ polkadot: 10 DOT :polkadot }}{{ kusama: 1 KSM :kusama }}, Dave tips
+{{ polkadot: 30 DOT :polkadot }}{{ kusama: 3 KSM :kusama }}, and Eve tips
 {{ polkadot: 100 DOT :polkadot }}{{ kusama: 10 KSM :kusama }}.
 
 The tip could have been closed out with only two of the three tippers. Once more than half of the
 tippers group have issued tip valuations, the countdown to close the tip will begin. In this case,
 the third tipper issued their tip before the end of the closing period, so all three were able to
 make their tip valuations known.
 
-Now the actual tip that will be paid out to Bob is the median of these tips, so Bob will be paid out
-{{ polkadot: 30 DOT :polkadot }}{{ kusama: 3 KSM :kusama }} from the Treasury.
-
-In order for Bob to be paid his tip, some account must call the `close_tip` extrinsic at the end of
+The actual tip that will be paid out to Bob is the median of these tips, so Bob will be paid out
+{{ polkadot: 30 DOT :polkadot }}{{ kusama: 3 KSM :kusama }} from the Treasury. In order for Bob to be paid his tip, some account must call the `close_tip` extrinsic at the end of
```

Comment:
Everything looks good to me!  

Have you experimented with any of the formatting commands:

```js
// Run on entire project
npm pretty-quick
// Run only on staged changes
npx pretty-quick --staged
// Run only on local changes
npx pretty-quick --branch
```

They will automatically try to reformat the newlines.  My understanding is that is was implemented to maintain better readable in text editors that don't wrap newlines.  Beware though these commands can make a huge amount of changes in files that haven't been run before.  I sometimes find it useful to run the command after the content is reviewed as to not make the review content overwhelming.

No action required, just something to test out if you haven't already.

Reply:
Run prettify on the branch, no changes.